shops/src/main/java/me/mykindos/betterpvp/shops/auctionhouse/menu/buttons/SubmitListingButton.java

```diff
@@ -62,15 +62,11 @@ public void handleClick(@NotNull ClickType clickType, @NotNull Player player, @N
 
         player.closeInventory();
 
-        if(UtilInventory.remove(player, auction.getItemStack())) {
-            auction.setExpiryTime(System.currentTimeMillis() + auction.getListingDuration().getDuration());
-            auctionManager.getAuctionRepository().save(auction);
-            auctionManager.getActiveAuctions().add(auction);
+        if (UtilInventory.remove(player, auction.getItemStack())) {
+            auctionManager.addNewAuction(player, auction);
         }
 
 
-        UtilMessage.simpleMessage(player, "Auction House", "Listing created successfully.");
```

Comment:
Add this back

Reply:
Never removed https://github.com/Mykindos/BetterPvP/blob/3a5ac73b1473a32435275ef3ed99bbbdfb6a935c/shops/src/main/java/me/mykindos/betterpvp/shops/auctionhouse/AuctionManager.java#L65-L64https://github.com/Mykindos/BetterPvP/blob/3a5ac73b1473a32435275ef3ed99bbbdfb6a935c/shops/src/main/java/me/mykindos/betterpvp/shops/auctionhouse/AuctionManager.java#L65-L64